Template/Tag/TawkToTag.php

```diff
@@ -35,6 +35,12 @@ public function getParameters() {
                 $field->validators[] = new NotEmpty();
                 $field->validators[] = new CharacterLength(16, 30); // we limit to 30 so users don't accidentally enter a 32 digit API key
             }),
+            $this->makeSetting('tawkToWidgetId', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'tawk.to Widget ID';
+                $field->description = 'You can get the Widget ID by logging into Tawk.to, going to "Administration" and clicking on "Chat Widget" and selecting an appropriate widget. The widget ID can be retrieved from "Direct Chat Link" as https://tawk.to/chat/{SITE_ID}/{WIDGET_ID}.';
+                $field->validators[] = new NotEmpty();
+                $field->validators[] = new CharacterLength(9, 20); // we limit to 20 so users don't accidentally enter a 32 digit API key
```

Comment:
@tsteur 
Couldn't find much on the length of widget ID
I tried setting the widget name to a very long text and the id was of length:9 only for safety I kept it till 20

Reply:
Also please check the description if the user can understand it or not

Reply:
@tsteur I was wondering should we make the description same for both the fields ? As we can get both the details from 1 link